apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -64,4 +66,37 @@ export const folderRouter = router({
         parentId,
       }
     }),
+  editFolder: protectedProcedure
+    .input(editFolderSchema)
+    .mutation(async ({ input: { resourceId, permalink, title, siteId } }) => {
+      if (!title && !permalink) {
+        return
+      }
```

Comment:
would this ever happen? we should not need to do anymore validation since we have a zod schema validating input right? if it's not meant to happen maybe we should prevent frontend from submitting this in the first place.

if you need a schema where you must have one of the two fields, we could update the schema:

```ts
z.object({
  resourceId: z.string(),
  permalink: z.string().optional(),
  title: z.string().optional(),
  siteId: z.string(),
})
  .superRefine((data, ctx) => {
  if (!data.permalink && !data.title) {
    ctx.addIssue({
      code: z.ZodIssueCode.custom,
      path: ["siteId"],
      message: "Either permalink or title must be provided.",
    })
    ctx.addIssue({
      code: z.ZodIssueCode.custom,
      path: ["title"],
      message: "Either permalink or title must be provided.",
    })
  }
})
```

Reply:
tbh i didn't go down this route because `superRefine` returns an error code and i was thinking this can just be a no-op (if you never give me anything to update, just skip updating)

not very opiniated here - i'm happy to make this change!